aws/aws.go

```diff
@@ -21,11 +23,13 @@ import (
 type S3API s3iface.S3API
 type LambdaAPI lambdaiface.LambdaAPI
 type SFNAPI sfniface.SFNAPI
+type DynamoDBAPI dynamodbiface.DynamoDBAPI
```

Comment:
I realize that the surrounding context does this, but as a rule it's better not to define new types when new types aren't needed (which they would be if you were adding methods to them, for instance). You could just use `dynamodbiface.DynamoDBAPI` throughout.

It would be better if that name [avoided stutter](https://golang.org/doc/effective_go.html#package-names) - if it were something more compact, like `dynamodbiface.API`. Since it's not, the desire for a shorter name is understandable, but then a [type alias](https://golang.org/ref/spec#Type_declarations) should suffice - and also make it clearer that this type isn't for any other purpose.

```
type DynamoDBAPI = dynamodbiface.DynamoDBAPI
```